frontend/projects/upgrade/src/app/shared/services/dialog.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DialogService } from './dialog.service';
+
+describe('DialogService', () => {
```

Comment:
if we aren't going to write tests, we should either delete this file or mark it `xdescribe` for now